php/fomo/composer.json

```diff
@@ -16,7 +16,7 @@
   ],
   "require": {
     "php": ">=8.1",
```

Comment:
```suggestion
```

not needed

PS : the idea is to keep the minimum amount of code for a working implementation (here this line is more an indication, we are on php8.2)

Reply:
There will be no problem with version 8.2